core-metrics/src/main/java/org/apache/kylin/metrics/lib/impl/BlockingReservoir.java

```diff
@@ -60,6 +60,10 @@ public BlockingReservoir(int minReportSize, int maxReportSize) {
     }
 
     public BlockingReservoir(int minReportSize, int maxReportSize, int maxReportTime) {
+        this(minReportSize, maxReportSize, maxReportSize, MAX_QUEUE_SIZE);
```

Comment:
it should be 
{code}
this(minReportSize, maxReportSize, maxReportTime, MAX_QUEUE_SIZE);
{code}

Reply:
Thank you for all your comments, I will raise a PR to fix them.